service/application/src/test/java/com/solace/maas/ep/runtime/agent/messagingServices/RtoMessagingServiceTests.java

```diff
@@ -34,7 +35,7 @@ public class RtoMessagingServiceTests {
     @Mock
     SessionEventCallback sessionEventCallback;
 
-
+    @Ignore
```

Comment:
Some M1 macs cannot execute this test so @Ignore since JavaRTO is not currently supported.